tests/dragonfly/requirements.txt

```diff
@@ -26,3 +26,4 @@ pytest-icdiff==0.8
 pytest-timeout==2.2.0
 asyncio==3.4.3
 fakeredis[json]==2.26.2
+hiredis==2.4.0
```

Comment:
Throwing this here, hiredis 3.0 is incompatible with redis 5.2 (latest release). 

TODO bump up the versions once they are fixed

Reply:
More on this https://github.com/redis/hiredis-py/pull/189 
and 
https://github.com/redis/redis-py/issues/3358#issuecomment-2553434973